server/utilities/smart-turn.mdx

```diff
@@ -220,7 +285,26 @@ To use the `LocalCoreMLSmartTurnAnalyzer`, you need to set up the CoreML model l
    export LOCAL_SMART_TURN_MODEL_PATH=/path/to/smart-turn
    ```
 
-Note that the CoreML model is optimized for Apple Silicon devices. If you're using a different platform, the remote Smart Turn service might be a better option.
+Note that the CoreML model is optimized for Apple Silicon devices. If you're using a different platform, consider using the PyTorch-based `LocalSmartTurnAnalyzer` or the remote Smart Turn service.
+
+### PyTorch Model Setup
+
+For the PyTorch-based `LocalSmartTurnAnalyzer`, you have two options:
+
+1. **Automatic Download**: If you don't specify a `smart_turn_model_path`, the model will be automatically downloaded from Hugging Face.
+
+2. **Local Model**: You can download the model to use it locally:
+
+   ```bash
+   # Install the Hugging Face CLI
+   pip install huggingface_hub
+
+   # Download the model
+   huggingface-cli download pipecat-ai/smart-turn --local-dir ./smart-turn
+
+   # Set the environment variable
+   export LOCAL_SMART_TURN_MODEL_PATH=./smart-turn
+   ```
```

Comment:
@markbackman, I haven't tried downloading the local model that way. I've only tried downloading it the same way we discussed for CoreML.